frontend/www/js/omegaup/components/arena/RunsForCourses.vue

```diff
@@ -243,215 +243,226 @@
-                </acronym>
-              </td>
-              <td>{{ run.language }}</td>
-              <td
-                v-if="showUser"
-                class="text-break-all text-nowrap"
-                :data-username="run.username"
-              >
-                <omegaup-user-username
-                  :classname="run.classname"
-                  :username="run.username"
-                  :country="run.country"
-                  :linkify="true"
-                  :href="'#runs'"
-                  :emit-click-event="true"
-                  @click="
-                    (username) =>
-                      (filterUsername = { key: username, value: username })
-                  "
-                ></omegaup-user-username>
-                <a :href="`/profile/${run.username}/`" class="ml-2">
-                  <font-awesome-icon :icon="['fas', 'external-link-alt']" />
-                </a>
-              </td>
-              <td v-if="showContest" class="text-break-all">
-                <a
-                  href="#runs"
-                  @click="
-                    onEmitFilterChanged({
-                      filter: 'contest',
-                      value: run.contest_alias,
-                    })
-                  "
-                  >{{ run.contest_alias }}</a
-                >
-                <a
-                  v-if="run.contest_alias"
-                  :href="`/arena/${run.contest_alias}/`"
-                  class="ml-2"
-                >
-                  <font-awesome-icon :icon="['fas', 'external-link-alt']" />
-                </a>
-              </td>
-              <td v-if="showProblem" class="text-break-all">
-                <a
-                  href="#runs"
-                  @click="
-                    onEmitFilterChanged({
-                      filter: 'problem',
-                      value: run.alias,
-                    })
-                  "
-                  >{{ run.alias }}</a
-                >
-                <a :href="`/arena/problem/${run.alias}/`" class="ml-2">
-                  <font-awesome-icon :icon="['fas', 'external-link-alt']" />
-                </a>
-              </td>
-              <td
-                :class="statusClass(run)"
-                data-run-status
-                class="text-center opacity-4 font-weight-bold"
-                hidden
-              >
-                <span class="mr-1">{{ status(run) }}</span>
-                <button
-                  v-if="!!statusHelp(run)"
-                  type="button"
-                  :data-content="statusHelp(run)"
-                  data-toggle="popover"
-                  data-trigger="focus"
-                  class="btn-outline-dark btn-sm"
-                  @click="showVerdictHelp"
+              <template v-if="run.status !== 'ready'">
+                <td colspan="16">
+                  <div class="line"></div>
+                </td>
+              </template>
```

Comment:
Here you don't need a template because you are using a single element, so it can work like this:
```suggestion
              <td v-if="run.status !== 'ready'" colspan="16">
                <div class="line"></div>
              </td>
```